change/@ni-eslint-config-angular-718354fd-7ab1-4395-ac53-e8505b9622bf.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Comment:
>Ran npm run print-available-rules and the only new, unset rule available in the updated version is @angular-eslint/runtime-localize (which we implemented), and that can be turned on as needed.

I would like to turn on the rule by default in the styleguide and to consider that within scope of the Angular upgrade feature if possible. We could either turn it on in this PR or in a follow up. Either way we'd want to communicate the change at UI WG so that client teams can fix violations (or, less ideally, temporarily disable the rule).

If you don't think you'll get to it in this feature, we should discuss how to make it happen afterward.

Reply:
I've turned the rule on in eslint-config-angular. We can make fixing any issues in clients part of the work to migrate them to Angular 18.